docs/INSTALLING.md

```diff
@@ -41,6 +42,21 @@ docker-compose build
 docker-compose up
 ```
 
+TROUBLESHOOTING:
```

Comment:
I would move this under:

> You can view more detailed

And add in `####` for `Troubleshooting`, without the caps-case and the colon.